wicketd/src/http_entrypoints.rs

```diff
@@ -286,13 +294,13 @@ async fn post_start_update(
 
     let opts = opts.into_inner();
     match opts.test_error {
-        Some(UpdateTestError::StartFailed) => {
+        Some(UpdateTestError::Fail) => {
             return Err(HttpError::for_bad_request(
                 None,
                 "Simulated failure while starting update".into(),
             ));
         }
-        Some(UpdateTestError::StartTimeout { secs }) => {
+        Some(UpdateTestError::Timeout { secs }) => {
             slog::info!(log, "Simulating timeout while starting update");
             // 15 seconds should be enough to cause a timeout.
             tokio::time::sleep(Duration::from_secs(secs)).await;
```

Comment:
I realize this was here before, but I think we probably want to either return an http error at this point ("we expected you to time out by now"?) or rename `Timeout` to something like `DelayResponse` to indicate that we will still respond, just after the specified number of seconds of delay.

Reply:
I added an error with a notice that this shouldn't be hit.